src/util/notify.cpp

```diff
@@ -52,6 +52,37 @@ namespace big::notify
 		}
 	}
 
+	void cage_blocked(CNetGamePlayer* player)
+	{
+		if (player)
+		{
+			if ((g_player_service->get_by_id(player->m_player_id)->is_friend() && g.session.trust_friends)
+			    || g_player_service->get_by_id(player->m_player_id)->is_trusted || g.session.trust_session)
+				return;
+
+			if (g.reactions.cage.notify)
+				g_notification_service.push_error("Protections", std::format("Blocked cage from {}", player->get_name()));
+
+			if (g.reactions.cage.log)
+				LOG(WARNING) << "Blocked cage from " << player->get_name() << " ("
+				             << (player->get_net_data() ? player->get_net_data()->m_gamer_handle.m_rockstar_id : 0) << ")";
+
+			if (g.reactions.cage.announce_in_chat)
+			{
+				auto msg = std::format("{} Blocked a cage from {}", g.session.chat_output_prefix, player->get_name());
```

Comment:
This string should be moved to the translation repository, this is a blocking requirement before this PR can be merged.

Reply:
Should i do the same for the crash method? As i see hard coded english strings and while im at it can do it as well

Reply:
LOG statements should remain in English if that's what you're asking.

Reply:
> LOG statements should remain in English if that's what you're asking.
> 
> Nvm, I see what you're saying. If you could add those to the translations as well that would be great.

Sounds good i will have a quick scout over the project files and add as many as i can after i have something to eat lol